bats/fb-katello-content.bats

```diff
@@ -89,6 +106,21 @@ setup() {
     --path="/tmp/stbenjam-dummy-0.2.0.tar.gz" | grep -q "Successfully uploaded"
 }
 
+@test "upload ostree_ref" {
+  if ! tIsRHEL 8; then
+    skip "OSTree content is only applicable on EL 8 systems"
+  fi
+
+  tSkipIfOlderThan43
+
+  wget --no-parent -r https://fixtures.pulpproject.org/ostree/small/
+  tFileExists fixtures.pulpproject.org/ostree && \
+    tar --exclude="index.html" -cvf "fixtures_small_repo.tar" -C fixtures.pulpproject.org/ostree "small"
+    hammer repository upload-content --name ${OSTREE_REPOSITORY} --content-type ostree_ref \
+      --path fixtures_small_repo.tar file --ostree-repository-name small
```

Comment:
I think the upload command is still malformed. Perhaps where it says `file`?

```
   Could not upload the content:
     Error: Too many arguments.
     
     See: 'hammer repository upload-content --help'.
```

Reply:
Perhaps you can run hammer in debug mode? I'm not sure if it has that, but this feels like an internal hammer issue.

Reply:
`file` should not be there at all.

Reply:
I think we also need the `--product` and `--organization` arguments.  The path may also need to be moved to /tmp/ because I'm not sure a relative path works with content uploads.